doc/manual/source/command-ref/nix-channel.md

```diff
@@ -53,6 +53,11 @@ This command has the following operations:
   Download the Nix expressions of subscribed channels and create a new generation.
   Update all channels if none is specified, and only those included in *names* otherwise.
 
+  > **Note**
+  >
+  > `--update` uses [fetchTarball](@docroot@/language/builtins.md#builtins-fetchTarball) under the hood, so it will cache channels.
```

Comment:
```suggestion
  > Downloaded channel contents are cached.
```

I don't think it matters how they're downloaded, and it's a distraction to point people away from the relevant subject, namely how to control the caching.